portal-wire-protocol.md

```diff
@@ -0,0 +1,161 @@
+# Portal Wire Protocol
+
+The Portal wire protocol is the base p2p protocol by which nodes communicate.
+
+The different Portal networks **MAY** use this base protocol, but they **MUST** remain separated per network.
+This is done at the [Node Discovery Protocol v5](https://github.com/ethereum/devp2p/blob/master/discv5/discv5-wire.md#talkreq-request-0x05) layer, by providing a different protocol bytestring, per network, in the `TALKREQ` message.
```

Comment:
When we use any of the `discv5_` json-rpc endpoints, we would want to send "standard" `TALKREQ` message via underlying discv5 protocol. What would be the protocol-id value of the message data? Where we would like to specify this?

Reply:
> Not sure I fully understand the question. I'll assume you are talking about the `sendTalkRequest` JSON-RPC method? 

I'm talking about all `discv5_*` endpoints defined for example in [this](https://playground.open-rpc.org/?schemaUrl=https://raw.githubusercontent.com/ogenev/portal-network-specs/jsonrpc-assembled-specs/jsonrpc-specs/openrpc.json&uiSchema%5BappBar%5D%5Bui:splitView%5D=false&uiSchema%5BappBar%5D%5Bui:input%5D=false&uiSchema%5BappBar%5D%5Bui:examplesDropdown%5D=false) json-roc draft specs. I guess those endpoints will be part of the production release and probably we want to specify protocol-id bytes for "base" discv5 messages? If `portal-*` is set as overlay protocol-id,  probably it makes sense to specify for example `discv5` protocol-id for underlying discv5 message calls?